wasm-engine/Cargo.toml

```diff
@@ -11,25 +11,27 @@ crate-type = ["cdylib", "rlib"]
 default = ["console_error_panic_hook"]
 
 [dependencies]
-getrandom = { version = "0.2", features = ["js"] }
-unleash-types = "0.10.5"
+wasm-bindgen = "0.2"
 unleash-yggdrasil = {path = "../unleash-yggdrasil"}
-serde-wasm-bindgen = "0.4.5"
-wasm-bindgen = "0.2.83"
+getrandom = { version = "0.2", features = ["js"] }
 
 # The `console_error_panic_hook` crate provides better debugging of panics by
 # logging them with `console.error`. This is great for development, but requires
 # all the `std::fmt` and `std::panicking` infrastructure, so isn't great for
 # code size when deploying.
-console_error_panic_hook = {version = "0.1.6", optional = true}
+console_error_panic_hook = { version = "0.1.1", optional = true }
```

Comment:
Why the downgrade here?

Reply:
Good spot, no good reason, this was modified by a tool, I've reverted it to the previous version